### PR TITLE
Since last steps

### PR DESCRIPTION
**Pull request by DonBraulio**

### CODE REVIEW

File: temporian/core/operators/since_last.py [hunk 1]
```diff
@@ -82,9 +97,12 @@ def has_sampling(self) -> bool:
 def since_last(
     input: EventSetOrNode,
     sampling: Optional[EventSetOrNode] = None,
+    steps: int = 1,
```
Comment (achoum) [inline temporian/core/operators/since_last.py:100]: The default value (1) is defined in several places. What about only defining it in the Eventset::since_last ?

File: temporian/implementation/numpy_cc/operators/since_last.cc [hunk 2]
```diff
@@ -8,46 +8,56 @@
...
-    while (next_event_idx < n_event && v_event[next_event_idx] <= t) {
-      next_event_idx++;
-    }
-    double value;
-    if (next_event_idx == 0) {
-      value = std::numeric_limits<double>::quiet_NaN();
-    } else {
-      value = t - v_event[next_event_idx - 1];
+namespace
+{
```
Comment (achoum) [inline temporian/implementation/numpy_cc/operators/since_last.cc:12]: The formatting has changed. It looks like a Microsoft c++ formatting.
Can you change it go Google c++ formatting (setting in VSCode)?

I'll look at the code after.

**Review by achoum — COMMENTED** [review]
Initial

File: temporian/implementation/numpy_cc/operators/since_last.cc (hunk 2 above)
Comment (DonBraulio) [reply temporian/implementation/numpy_cc/operators/since_last.cc]: Fixed 👍

File: temporian/core/operators/since_last.py (hunk 1 above)
Comment (DonBraulio) [reply temporian/core/operators/since_last.py]: Done.

File: temporian/core/operators/since_last.py [hunk 3]
```diff
@@ -32,10 +32,14 @@ class SinceLast(Operator):
     def __init__(
         self,
         input: EventSetNode,
+        steps: int,
         sampling: Optional[EventSetNode] = None,
     ):
         super().__init__()
 
+        if steps <= 0:
+            raise ValueError("Invalid value received: {steps=}")
```
Comment (ianspektor) [inline temporian/core/operators/since_last.py:41]: would be nice to specify in the error that steps need to be >=1

**Review by ianspektor — APPROVED** [review]

File: temporian/core/operators/since_last.py (hunk 3 above)
Comment (DonBraulio) [reply temporian/core/operators/since_last.py:41]: Totally 😄

**Pull request merged**